hudi-common/src/main/java/org/apache/hudi/common/util/RetryHelper.java

```diff
@@ -47,13 +48,24 @@ public RetryHelper(long maxRetryIntervalMs, int maxRetryNumbers, long initialRet
     if (StringUtils.isNullOrEmpty(retryExceptions)) {
       this.retryExceptionsClasses = new ArrayList<>();
     } else {
-      this.retryExceptionsClasses = Arrays.stream(retryExceptions.split(","))
-          .map(exception -> (Exception) ReflectionUtils.loadClass(exception, ""))
-          .map(Exception::getClass)
-          .collect(Collectors.toList());
+      try {
+        this.retryExceptionsClasses = Arrays.stream(retryExceptions.split(","))
+                .map(exception -> (Exception) ReflectionUtils.loadClass(exception, ""))
+                .map(Exception::getClass)
+                .collect(Collectors.toList());
+      } catch (HoodieException e) {
```

Comment:
Why this change ? If there is no default clazz for retryExceptions, should skip this construction ~

Reply:
If the filled `Exception class name` does not exist,`java.lang.ClassNotFoundException` a exception will be thrown.